pkg/service/schema.go

```diff
@@ -278,9 +304,17 @@ func (f FloatPropertySchema) validate(context string, value interface{}) error {
 	if value == nil {
 		return nil
 	}
-	val, ok := value.(float64)
-	if !ok {
-		return NewValidationError(context, "field value is not of type float64")
+	var val float64
+	if floatVal, ok := value.(float64); ok {
+		val = floatVal
+	} else if floatVal, ok := value.(*float64); ok {
+		val = *floatVal
+	} else if floatVal, ok := value.(float32); ok {
+		val = float64(floatVal)
+	} else if floatVal, ok := value.(*float32); ok {
+		val = float64(*floatVal)
+	} else {
+		return NewValidationError(context, "field value is not of type float")
```

Comment:
All of the above just makes validations more tolerant of the types in the map that might represent a float.